package.json

```diff
@@ -119,6 +129,96 @@
         "command": "extension.vim_down",
         "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !parameterHintsVisible"
       },
+      {
+        "key": "shift+left",
+        "command": "extension.vim_shift+left",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "shift+right",
+        "command": "extension.vim_shift+right",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "shift+up",
+        "command": "extension.vim_shift+up",
+        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !parameterHintsVisible"
+      },
+      {
+        "key": "shift+down",
+        "command": "extension.vim_shift+down",
+        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !parameterHintsVisible"
+      },
+      {
+        "key": "shift+home",
+        "command": "extension.vim_shift+home",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "shift+end",
+        "command": "extension.vim_shift+end",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "shift+pageup",
+        "command": "extension.vim_shift+pageup",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "shift+pagedown",
+        "command": "extension.vim_shift+pagedown",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "ctrl+left",
+        "command": "extension.vim_ctrl+left",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "ctrl+right",
+        "command": "extension.vim_ctrl+right",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "ctrl+shift+left",
+        "command": "extension.vim_ctrl+shift+left",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "ctrl+shift+right",
+        "command": "extension.vim_ctrl+shift+right",
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
+      },
+      {
+        "key": "ctrl+shift+up",
+        "command": "extension.vim_ctrl+shift+up",
+        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !parameterHintsVisible"
+      },
+      {
+        "key": "ctrl+shift+down",
+        "command": "extension.vim_ctrl+shift+down",
+        "when": "editorTextFocus && vim.active && !inDebugRepl && !suggestWidgetVisible && !parameterHintsVisible"
+      },
+      {
+        "key": "ctrl+Home",
```

Comment:
I assume it doesn't matter, but is there a reason `Home` and `End` are capitalized here but not below?

Reply:
This was because it was copied from the `"key": "Home",` above. I'll make them all lowercase.